cdap-ui/app/directives/my-post-run-action-wizard/my-post-run-action-wizard-modal.less

```diff
@@ -47,3 +47,22 @@
     }
   }
 }
+body.theme-cdap.state-hydrator-detail {
+  .modal.hydrator-modal.node-config-modal {
+    &.post-run-actions-modal {
+      .modal-dialog {
+        .modal-content {
+          .modal-body {
+            height: unset;
+            max-height: ~"calc(100% - 165px)";
```

Comment:
Why `165px`? Should it be moved out to a constant?

Reply:
Have added comment for reference. It is to do with the height of the modal taking the height from html rather than body. We apply an extra 50px for margin bottom on the body element which is included here. This is not the ideal way to style the modal but I didn't want to introduce a major change at this point in the release. Hope the comment makes sense.